entities/tokens.go

```diff
@@ -0,0 +1,20 @@
+package entities
+
+import (
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type TokenField string
+
+const (
+	TokenID      TokenField = "_id"
+	TokenJWT     TokenField = "jwt"
+	TokenCreator TokenField = "creator"
+)
+
+// Token is the struct to store tokens
+type Token struct {
```

Comment:
Would it maybe make sense to call it `ServiceToken`? Since we're not going to be explicitly storing user tokens.

Reply:
Good call, that's a more reflective name 👍